packages/labs/context/src/lib/std-decorators/provide.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {ReactiveElement} from '@lit/reactive-element';
+import {Context} from '../create-context.js';
+import {ContextProvider} from '../controllers/context-provider.js';
+
+/*
+ * IMPORTANT: For compatibility with tsickle and the Closure JS compiler, all
+ * property decorators (but not class decorators) in this file that have
+ * an @ExportDecoratedItems annotation must be defined as a regular function,
+ * not an arrow function.
+ */
+
+/**
+ * A property decorator that adds a ContextConsumer controller to the component
```

Comment:
```suggestion
 * A property decorator that adds a ContextProvider controller to the component
```

Not sure fi this needs to be done in the existing decorators too